Documentation/LocalizedContent/Index.rst

```diff
@@ -28,105 +33,93 @@ ask you about this choice.
    The translation wizard asks for a choice of localization strategy
 
 
-The "Translate" button corresponds to the first way, the "Copy"
-button to the second way. Furthers details on how to proceed with
+The "Translate" button corresponds to the connected mode, the "Copy"
+button to the free mode. Furthers details on how to proceed with
 translations are found in the :ref:`Editor's Tutorial <t3editors:languages>`.
 
 
-.. _localized-content-bound:
+.. _localized-connected-content:
 
-Bound content
+Connected content
 """""""""""""
```

Comment:
Underline should at least be same length as title (longer is not a problem, but shorter may be).